core/harness.cpp

```diff
@@ -426,12 +437,17 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
 
     vector<Node> nodes = g.GetOrderedNodes();
     void *ref[nodes.size()][MAX_CONN];
+    memset(ref, 0xca, sizeof(ref));
 
     for (Node n : nodes) {
-        void *in_ref[n.in_ref_size()];
-        void *out_ref[n.out_ref_size()];
+        void *in_ref[n.in_ref_size() + 1];
```

Comment:
could you explain this part

Reply:
UBSan detects zero sized VLA on the stack as UB.